packages/workspace/src/browser/workspace-commands.ts

```diff
@@ -89,6 +89,7 @@ export namespace WorkspaceCommands {
         category: WORKSPACE_CATEGORY,
         label: 'Close Workspace'
     });
+    /** @deprecated @since 1.24 Use CommonCommands.NEW_FILE instead. */
     export const NEW_FILE = Command.toDefaultLocalizedCommand({
```

Comment:
I left this one for the context menu in the navigator, since we still don't have the ability to do input fields in trees as we should for that command.